macros/database_specific_helpers/type_helpers.sql

```diff
@@ -44,6 +48,21 @@
    array<string>
 {% endmacro %}
 
+{% macro redshift__type_array() %}
+   super
+{% endmacro %}
+
+{% macro type_string() %}
+   {{ return(adapter.dispatch('type_string', 'dbt_artifacts')()) }}
+{% endmacro %}
+
+{% macro default__type_string() %}
+   {{ return(api.Column.translate_type("string")) }}
+{% endmacro %}
+
+{% macro redshift__type_string() %}
+   varchar(max)
+   
```

Comment:
Missing endmacro block leading to a compilation error:
```suggestion
{% endmacro %}
```

Reply:
Thanks @shiv-io!